miso-web/src/main/webapp/scripts/sample_hot.js

```diff
@@ -953,126 +1067,117 @@ Sample.hot = {
     sample.projectId = (parseInt(obj.projectId) || parseInt(document.getElementById('projectSelect').value));
     sample.scientificName = obj.scientificName;
     if (obj.receivedDate && obj.receivedDate.length) {
-      sample.receivedDate = obj.receivedDate;
+      // the time string is added for detailedSample because the server is expecting a datetime value
+      sample.receivedDate = obj.receivedDate += "T00:00:00-05:00";
```

Comment:
Also in GLT-779, I had removed the hardcoded timezone from here. This should just be `sample.receivedDate = obj.receivedDate;`